apps/src/templates/census2017/YourSchool.jsx

```diff
@@ -27,25 +30,58 @@ const styles = {
   }
 };
 
-export default class YourSchool extends React.Component {
+export default class YourSchool extends Component {
   static propTypes = {
-    alertHeading: React.PropTypes.string,
-    alertText: React.PropTypes.string,
-    alertUrl: React.PropTypes.string,
-    hideMap: React.PropTypes.bool
+    alertHeading: PropTypes.string,
+    alertText: PropTypes.string,
+    alertUrl: PropTypes.string,
+    hideMap: PropTypes.bool
   };
 
+  constructor(props) {
+    super(props);
+    this.responsive = new Responsive();
+    this.state = {
+      windowWidth: $(window).width(),
+      windowHeight: $(window).height(),
+      mobileLayout: this.responsive.isResponsiveCategoryInactive('md')
+    };
+  }
+
   componentDidMount() {
     if (!this.props.hideMap) {
       $('#map').appendTo(ReactDOM.findDOMNode(this.refs.map)).show();
     }
+    // Resize handler.
+    window.addEventListener('resize', _.debounce(this.onResize, 100).bind(this));
+  }
+
+  onResize() {
+    const windowWidth = $(window).width();
+    const windowHeight = $(window).height();
+
+    // We fire window resize events when the grippy is dragged so that non-React
```

Comment:
These comments don't need to be here and were copied inadvertently.  (I should also remove them from https://github.com/code-dot-org/code-dot-org/blob/f1201c3962f9b668853b0a68c0464b52ea3ee387/apps/src/templates/studioHomepages/Courses.jsx#L55)